### PR TITLE
Create example mobile schema

### PR DESCRIPTION
**Pull request by rtravitz**
## Description of change

This is just a draft pull request for us to discuss the diffed changes identifying mobile facilities in the Open API schema. It is not intended to be merged.

### CODE REVIEW

File: modules/va_facilities/README.v1.yml [hunk 1]
```diff
@@ -37,6 +37,10 @@ info:
     - The list of available services in the `services` element is only applicable to VA health and
       benefits facilities.
 
+    ### Mobile Facilities
+
+    VA has a set of mobile facilities that move regularly within a region. If a facility comes back from this API with `"mobile": "true"`, the latitude and longitude should be considered inaccurate.
```
Comment (nfasulo) [inline modules/va_facilities/README.v1.yml]: I think we need to mention that the address should be considered inaccurate as well.

Comment (nfasulo) [inline modules/va_facilities/README.v1.yml]: Maybe we should also say that the appropriate way to get the current address is the contact the mobile facility directly.

**Comment by rtravitz** [comment]
> This makes me wonder if mobile facilities should be excluded from nearby and drive_time bands. If we can't trust the location of the facility are we doing more harm than good in telling someone that it is close to a veteran?

I agree with this. We should raise it back to the team!

**Comment by davem-va** [comment]
I know from enough DST discussions, that the DST team will very much want the mobile sites (and includes them today). 

I agree that they may be less useful than the regular sites for drive time assessment but given DST and that the CCE API returns the facilities meeting drive time criteria, the application has the means to include/excluded, assuming we add the "mobile" field to each facility in the /nearby endpoint and the CCE API.

**Pull request closed without merging**